mep/books/tests/test_books_views.py

```diff
@@ -153,17 +154,32 @@ def test_get_queryset(self):
         view.request = self.factory.get(self.url)
         # if form is valid, should return all works sorted by chosen sort
         form.is_valid.return_value = True
-        form.cleaned_data = {'sort': 'title'}
+        form.cleaned_data = {'sort': 'title', 'circulation_dates': ''}
         solr_qs = view.get_queryset()
         db_qs = Work.objects.order_by('sort_title')
+        print(db_qs)
```

Comment:
should this `print()` statement still be here?

Reply:
Nope! thank you for catching